src/components/AuthFields.jsx

```diff
@@ -3,22 +3,55 @@ import PropTypes from "prop-types";
 import PureTextInput from "./PureTextInput";
 
 export const AuthFields = ({ authType, onHeaderChange }) => {
+	const [userName, setUserName] = useState("");
```

Comment:
Let's rename this to `username` instead and update the method to `setUsername`. Update the same in all usages